### PR TITLE
Added tag support script

### PR DESCRIPTION
**Pull request by akru**
**Pull Request Summary**

This script could help tag management.

**Check list**
- [ ] contains breaking changes
- [ ] adds new feature
- [ ] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tests and/or benchmarks are included
- [ ] changed API client type definition or chain metadata

**This pull request makes the following changes:**

**Adds**
- (ex: Add feature A)

**Fixes**
- (ex: Fix validation function)

**Changes**
- (ex: Change document B)

**To-dos**
> *Feel free to remove this section if it's not applicable

- [ ] (ex: add user list)

### CODE REVIEW

File: scripts/make-tags.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        echo "$CARGO_TOML_PATH does not exist"
+        exit 1
+    fi
+
+    PKG_NAME=$(cat $CARGO_TOML_PATH | grep name | awk '{print $3}' | sed 's/"//g')
+    PKG_VERSION=$(cat $CARGO_TOML_PATH | grep version | awk '{print $3}' | sed 's/"//g')
+
+    local line key value entry_regex
+    entry_regex="^[[:blank:]]*([[:alnum:]_-]+)[[:blank:]]*=[[:blank:]]*('[^']+'|\"[^\"]+\"|[^#]+)"
+    while read -r line; do
```
Comment (Dinonard) [inline scripts/make-tags.sh:27]: Just curious - why do you need to parse the file here if you extracted the information in L25 and L26?

Comment (akru) [reply scripts/make-tags.sh:27]: We can't be sure it's on L25/L26, according to spec it could be any place

Comment (Dinonard) [reply scripts/make-tags.sh:27]: I meant the lines in your script - you extract both PGN_NAME and PKG_VERSION using grep.

**Comment by Dinonard** [comment]
@akru Can we go with this and merge it?
I'd like us to stop depending on active dev branch soon.

File: scripts/make-tags.sh (hunk 1 above)
Comment (akru) [reply scripts/make-tags.sh:27]: Yeah, my bad, fixed

**Pull request merged**